active_directory/assets/configuration/spec.yaml

```diff
@@ -4,18 +4,11 @@ files:
   options:
   - template: init_config
     options:
+    - template: init_config/perf_counters
     - template: init_config/default
   - template: instances
     options:
-    - template: instances/pdh_legacy
```

Comment:
I see in the other PRs similar to this one we do this too, but why don't we keep this template but make it hidden?